pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -189,12 +190,13 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
             if status in HIDE_STATUS:
                 continue
             if status == STATUS_ACCEPTED:
-                msg = "    A - Accepted (Standards Track only) or Active proposal"
+                msg = "* A - Accepted (Standards Track only) or Active proposal"
             else:
-                msg = f"    {status[0]} - {status} proposal"
+                msg = f"* {status[0]} - {status} proposal"
             self.emit_text(msg)
```

Comment:
```suggestion
            self.emit_text(
                f"* **{status_code}** --- *{status}*: ABBREVIATED_STATUSES[status]")
```

Also, needs to import `ABBREVIATED_STATUSES` from `pep_headers` (which is possible now that it is a module-level dict constant :) )